R/polish_sector_profile_product.R

```diff
@@ -11,7 +11,8 @@ polish_sector_profile_product <- function(sp_prod, europages_companies, ecoinven
   prepare_inter_sector_profile(sp_prod, europages_companies, ecoinvent_activities, ecoinvent_europages, isic) |>
     relocate_sector_profile_product() |>
     rename_sector_profile_product() |>
-    mutate(scenario = recode(.data$scenario, "1.5c rps" = "IPR 1.5c RPS", "nz 2050" = "WEO NZ 2050")) |>
+    mutate(scenario = ifelse(is.na(scenario), grouped_by, scenario)) |>
+    mutate(scenario = recode_scenario(.data$scenario)) |>
```

Comment:
Here I write a custom helper to recode scenario in a more robust way. It may be wrong but the idea may be useful. I'll discuss in the helper itself.